config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: %i[new create show]
+  get 'sessions/new', to: 'sessions#new' 
+  post 'sessions', to: 'sessions#create'
+  delete 'sessions', to: 'sessions#destroy'
+  # EventsController routes
+  get 'events/new', to: 'events#new'
+  post 'events/new', to: 'events#create'
+  get 'events', to: 'events#index'
+  get 'events/:id', to: 'events#show'
+end
```

Comment:
# OPTIONAL CHANGES ♻️ 

### Invites feature \[EXTRA CREDIT IN ODIN\]

* [ ] route under `/event/:event_id/invites/new`
* [ ] Invite model `belongs_to :event`
* [ ] Invite model `belongs_to :user`
* [ ] User model `has_many :invitations`
* [ ] Event model `has_many :invitations`
* [ ] Join model (EventAttendance) is required to make sure that atenndance is not created without invitation (e.g. `before_create` callback usage)